cmd/assets/CORONA EXTRA.json

```
@@ -0,0 +1,245 @@
+        "proportion": {
+          "Width": 0,
+          "Height": 0
+        }
+      }
+    ],
+    "09": [
+      {
+        "value": "ОБВОДНОГО КАНАЛА, Д. 21, ЛИТЕР А, ПОМ. 1-Н/1, ЭТАЖ 1, ТЕЛ.: +7 (812) 429-39-46, E-MAIL: LUCATE-IMPORT@MAIL.RU;",
+		"style": "B",
+        "font_size": 3.5,
+        "align": "L",
```

Biome (2.1.2)

[error] The key style was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)